src/tools/web/elektrad/README.md

```diff
@@ -1,31 +1,63 @@
-# elektra-web/elektrad
+# Elektrad
 
-A server that provides an [HTTP API](http://docs.elektrad.apiary.io/) to access
-Elektra remotely, built using [express](https://expressjs.com/).
+## Introduction
 
-## Installation
+A server that provides an [HTTP API](http://docs.elektrad.apiary.io) to access
+Elektra remotely, built using [Go](https://golang.org).
 
-- first, install [libelektra](https://libelektra.org/)
-- now run `npm install`
+## Lifetime of KDB handles
 
-## Running
+Instantiating a KDB Handle for every request is expensive, espescially for big KDB databases, and prevents handling of conflicts. To mitigate this issue sessions with an associated handle are created. One hour after the last request these sessions are destroyed and the KDB handle is closed.
```

Comment:
Can this be abused for denial of service or similar? What about having some MAX_SESSIONS limit and removing the oldest sessions to mitigate the problems? It is not relevant for this PR though.

Reply:
Good idea, would you open up an issue for that?

Reply:
See #3360.